pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -459,18 +460,27 @@ func TestGetPipelineFunc_RemoteResolutionInvalidData(t *testing.T) {
 }
 
 func TestGetVerifiedPipelineFunc_Success(t *testing.T) {
+	// This test case tests the success cases of trusted-resources-verification-no-match-policy when it is set to
+	// fail: passed matching policy verification
+	// warn and ignore: no matching policies.
 	ctx := context.Background()
+	signer, _, k8sclient, vps := test.SetupVerificationPolicies(t)
```

Comment:
This issue wasn't introduced by this PR, but having the test data separated into a helper function for creation (e.g. having a helper function create verification policies) makes it harder to understand what's being tested without having to jump to the helper function.

Reply:
Noted! I will see if I can refactoring those test helpers into those pkgs, there's an issue tracking this but it is not of high priority

Reply:
thanks! what's the issue?

Reply:
this one: https://github.com/tektoncd/pipeline/issues/5820

Reply:
This issue looks a bit unrelated; could you add a bit more detail in a comment re: moving test data closer to test cases?

Reply:
Oh I see, I have updated the issue to "refactor the helper functions to the files where they are used"